### ISSUE TITLE
Feature Request: Pass commit message to post-code-update hook

### ISSUE THREAD

**Issue by shawnachieve** [acquia/cloud-hooks#27]
It would be useful to have the last git commit message passed to the post-code-update hook.

The reason for this is that we have a series of fairly resource intensive update steps that occur with every commit.  If we had access to the commit message though we could parse that to determine which updates are necessary.

For example, if we're just pushing a CSS update to the site then there's no need to re-run content synchronization steps, update db, or features revert.

**Comment by mhumeSF** [comment]
:+1:

**Comment by richardbporter** [comment]
+1

**Comment by joeparsons** [comment]
Yes, please!

**Comment by bkosborne** [comment]
Yes, we need this as well for the same reasons. There's really no way to get the most recent commit message either because the repo is not deployed on the servers.

**Comment by StevenGFX** [comment]
Would love to have this as well

**Comment by Loguercio** [comment]
I'd also like to be able to do this. 

@shawnachieve did you ever work out a way to skip updates?

**Comment by shawnachieve** [comment]
@Loguercio Unfortunately, I did not find a way to make this work.

**Comment by Loguercio** [comment]
@shawnachieve fyi I just got it working last night. I ended up using travis to read the last commit message and then create a flag file in the deployment artifact. I then had the cloud hook check that file to determine whether to run the site updates or not. It's a little messy, but with 50+ multisites (and growing!) it'll save us hours each deployment.

**Comment by bkosborne** [comment]
Acquia... please, add this. I'm begging!
